packages/ppx/src/css_to_emotion.re

```diff
@@ -509,13 +483,18 @@ let render_keyframes = (declarations: rule_list) => {
 };
 
 let render_global = ((ruleList, loc): stylesheet) => {
-  ruleList
-  |> List.filter_map(rule => {
-       switch (rule) {
-       | Style_rule(rule) => Some(rule)
-       | _ => None
-       }
-     })
-  |> render_style_rules(~loc, CssJs.global(~loc))
+  let styles =
+    ruleList
+    |> List.map(rule => {
+         switch (rule) {
+         | Style_rule(style_rule) => render_style_rule(style_rule)
+         | At_rule(at_rule) => render_at_rule(at_rule)
+         | _ =>
+           Generate_lib.error(~loc, "only style rule and at rule supported")
```

Comment:
Nit: this error only raises when there's a declaration. Can you add an error message like: 

"declarations do not make sense in global, only style rules or @media/@print/etc are supported. If you want to apply styles to everything, try `*` {}"

(^ This is just a placeholder error message)